lib/ddtrace/contrib/rack/request_queue.rb

```diff
@@ -0,0 +1,33 @@
+module Datadog
+  module Contrib
+    module Rack
+      # QueueTime simply...
+      module QueueTime
+        REQUEST_START = 'HTTP_X_REQUEST_START'.freeze
+
+        module_function
+
+        def get_request_start(env, now = Time.now.utc)
+          header = env[REQUEST_START]
+
+          if header
```

Comment:
I usually prefer early returns than code nested in conditional branches, but that's only a matter of personal style :)

Reply:
👍